resources/assets/sass/_customscrollbar.scss

```diff
@@ -0,0 +1,1267 @@
+/*
```

Comment:
I think you should remove this file. You have already imported it.
https://github.com/timongorpg/timongo/pull/37/files#diff-1dee3aa6a79c9f1c3bf0abca2c6fd30fR12